wiki/en/Running-a-Server.md

```diff
@@ -52,7 +52,7 @@ Unless you plan on hosting more than about 5 players on a slower-speed home conn
 
 - Running a Private Server at home will require you to [port forward](#running-a-private-server) on your router. When running a Private Server, port forwarding should not be necessary in most cases, but it's advisable to do so because some networks may not work properly with Jamulus in its default mode.
 
-- Jamulus doesn't currently support IPv6
+- Jamulus only has limited IPv6 support which needs to be enabled with a command line argument on the client and server. There are plans to expand IPv6 support.
```

Comment:
Not sure in how much detail we want to go here

Reply:
I think what's there looks okay.  At this stage, anyone wanting to use the option needs to know how to go read the command line options page without being told exactly what to do.

Reply:
I've not been following the IPv6 ticket very closely, but is it worth saying "Jamulus only has limited IPv6 support _for private servers only_, which ..." - or is that not helpful?

Reply:
"private" and "public" and "central" got deprecate, so "no" :)

It could have "Server registration is currently not supported." added.

Reply:
```suggestion
- Jamulus only has limited IPv6 support which needs to be enabled with a command line argument on the client and server. There are plans to expand IPv6 support for connections started from the server list.
```